Apps/Sandcastle/gallery/Atmosphere.html

```diff
@@ -0,0 +1,710 @@
+<!DOCTYPE html>
```

Comment:
Thoughts on removing the [Sky Atmosphere](https://sandcastle.cesium.com/?src=Sky%20Atmosphere.html) demo and moving those imagery adjustment controls here?

Reply:
Sure, we can do that. I think we can consolidate the `GroundAtmosphere` here too. I'll have to make some adjustments to UI to make the list scrollable because it's not ideal for smaller screens.

Reply:
True I forgot about that one. Good idea to combine them all, maybe with collapsible panels or something.

Reply:
Added the options from the other two Sandcastles into this new one. Removed the other two. Added entry in CHANGES.md